src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -22,6 +22,7 @@ import scala.collection.immutable
 import scala.concurrent.duration._
 import scala.concurrent.{ExecutionContext, Future}
 
+case class ClusterNotReadyException(googleProject: GoogleProject, clusterName: String) extends LeoException(s"Cluster $googleProject/$clusterName is not ready yet, chill out and try again later", StatusCodes.EnhanceYourCalm)
```

Comment:
reminder: swagger

Reply:
Blergh right. Btw our swagger is full of 
```
        404:
          description: Cluster not found
          schema:
            $ref: '#/definitions/ErrorReport'
```
which I guess is not correct anymore.

Reply:
Er I guess we're still returning an ErrorReport, just no stack trace

Reply:
yeah, it's def an ErrorReport object. in the majority of cases the stacktrace will be empty but this isn't wrong